CONTRIBUTING.md

```diff
@@ -149,15 +151,11 @@ black manually to format files and commit them.
 
 To install `flake8`, `ufmt` and `mypy`, please run
 
-```bash
-bash ./tests/run_code_style.sh install
-```
-
 To format files and commit changes:
 
 ```bash
 # This should autoformat the files
-bash ./tests/run_code_style.sh fmt
+pre-commit -a
```

Comment:
We need to update better this file, just above the section is called "##### Formatting without pre-commit".

Reply:
Fixed